jepsen-flink/.gitignore

```diff
@@ -0,0 +1,17 @@
+*.class
+*.iml
+*.jar
+*.retry
+.DS_Store
+.hg/
+.hgignore
+.idea/
+/.lein-*
+/.nrepl-port
+/checkouts
+/classes
+/target
+pom.xml
+pom.xml.asc
+store
+bin/DataStreamAllroundTestProgram.jar
```

Comment:
Maybe we could ignore the complete `bin/` folder. I put, for example, my `flink-dist.tgz` there and git reports it now as an untracked file.

Reply:
Good point. I fixed it.